glib/GType.cs

```diff
@@ -69,6 +69,8 @@ public static GType FromName (string native_name)
 		static Dictionary<IntPtr, Type> types = new Dictionary<IntPtr, Type> (IntPtrEqualityComparer.Instance);
 		static Dictionary<Type, GType> gtypes = new Dictionary<Type, GType> ();
 
+		// TODO: Add ignore map for assemblies, so we can skip assemblies which don't need to be scanned for types.
+
```

Comment:
Do you want to merge a TODO?

Reply:
No, I'll remove it, it's not needed anymore.